plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -182,6 +182,7 @@
 public final class IcebergUtil
 {
     public static final String TRINO_TABLE_METADATA_INFO_VALID_FOR = "trino_table_metadata_info_valid_for";
+    public static final String TRINO_TOO_LONG_TABLE_COMMENTS = "trino_too_long_table_comments";
```

Comment:
I feel there is no need to include the reason (too long) to the property name. We may find different situations that we can't store table comments in the future. How about renaming to `trino_invalid_table_comments` or something? I don't come up with good naming, though.

Reply:
what about `TRINO_NOT_CACHEABLE_TABLE_COMMENT` ?